src/Chirp.Infrastructure/IAuthorRepository.cs

```diff
@@ -0,0 +1,11 @@
+namespace Chirp.Core
```

Comment:
What is the argument of having a double layered interface, since this and ICheepRepository are both only included in the ICheepService interface?
Or am I misssing something?